src/views/Home/index.view.tsx

```diff
@@ -22,9 +21,40 @@ const description = [
   {
     description:
       "CruzHacks is the largest hackathon in Santa Cruz. Each year, we invite hundreds of students to develop solutions to real-world problems, pursue inclusion in tech, and kindle the spirit of innovation.  ",
+    style: "description",
+    line: 0,
   },
   {
     description: "This year, we’re bringing CruzHacks to you.  ",
+    style: "description",
+    line: 1,
+  },
+  {
+    description: (
+      <>
+        <span className="Hero__details">
+          <a
+            className="Hero__details--link"
+            href="http://mlh.io/code-of-conduct"
+            target="_blank"
+            rel="noreferrer"
+          >
+            Our Code of Conduct
+          </a>
+
+          <a
+            className="Hero__details--link"
+            href="mailto:contact@cruzhacks.com"
+            target="_blank"
+            rel="noreferrer"
+          >
+            Contact Us
+          </a>
+        </span>
+      </>
```

Comment:
Is there a way to condense this code? To me it looks a bit repetitive & I think there can be a way to avoid adding an entire react component here.

Reply:
yea you're right, I was a bit lazy here & didn't bother to conditionally render links at the hero component, where this code should be at. I went ahead & turned this back to simple strings